frontend/app/utils/string-utils.ts

```diff
@@ -35,3 +35,13 @@ export const padWithZero = (value: number, maxLength: number) => {
   if (value.toString().length >= maxLength) return value.toString();
   return value.toString().padStart(maxLength, '0');
 };
+
+/**
+ *
+ * @param input - The string value of the input field to be validated
+ * @returns Boolean indicating whether the input contains all valid characters
+ */
+export function checkIfInputContainsValidCharacters(input: string): boolean {
```

Comment:
as i mentioned in slack, only `\u00a0` is valid for whitespace, according to the business requirements.

Reply:
added regular whitespace to the regex.  I guess we're just waiting on business to confirm.  The ticket you commented on in ADO hasn't had any activity since you posted.

Reply:
I wouldn't make sense to not have the regular space.